addons/web_tour/static/src/js/tip.js

```diff
@@ -315,35 +316,39 @@ var Tip = Widget.extend({
     },
     _bind_anchor_events: function () {
         this.consume_event = this.info.consumeEvent || Tip.getConsumeEventType(this.$anchor, this.info.run);
-        this.$consumeEventAnchor = this.$anchor;
+        this.$bubbleEventAnchors = this.$anchor;
+        if (this.$altAnchor) {
+            this.$bubbleEventAnchors = this.$bubbleEventAnchors.add(this.$altAnchor);
+        }
+        this.$consumeEventAnchor = this.$bubbleEventAnchors;
```

Comment:
You assume that both trigger and alt_trigger have the same consume event type. What if trigger is a button selector (consumed with 'click') and alt_trigger is an input (consumed with 'change') ?

Reply:
👍